packages/foreman/foreman-release/foreman-release.spec

```diff
@@ -0,0 +1,53 @@
+Name:     foreman-release
+Version:  1.19.0
+Release:  1%{?dist}
+
+Summary:  Foreman repositories meta-package
+Group:    Applications/System
+License:  GPLv3+
+URL:      https://theforeman.org
+Source0:  foreman.repo
+Source1:  foreman-plugins.repo
+Source2:  foreman.gpg
+Source3:  tfm-ror51.repo
```

Comment:
In other places I've named this `foreman-rails.repo` now which means less changes when moving to ror52.